core/javaAction/init.py

```diff
@@ -18,12 +15,12 @@
     encoded = sys.argv[1]
 
 payload = {
```

Comment:
btw, https://github.com/openwhisk/openwhisk/blob/master/core/actionProxy/invoke.py#L66 should be >= 1 (mind fixing at the same time?)